src/modules/combinedFacets/CombinedFacetsContainer.js

```diff
@@ -15,6 +18,24 @@ import GenerateCombinedFacets from './generateCombinedFacets';
 import { executeCallback } from '../../common/utils';
 
 class CombinedFacetsContainer extends React.PureComponent {
+
+    componentDidMount(){
+        const {
+            categoryDisplayName = '',
```

Comment:
Are we will using these variables?

Reply:
its used in the method setMultilevelFacetsConfiguration

Reply:
But do we require it here?